window_doc.talon

```diff
@@ -0,0 +1,6 @@
+os: mac
+--
+
+open current (file|directory|folder|document): user.open_current_doc()
+reveal current (file|directory|folder|document): user.reveal_current_doc()
```

Comment:
noun verb — just "document reveal" or maybe "document show" seems fine to me

Reply:
Thanks, done. I kept "reveal" as an alias, I feel like that has specific macOS meaning

Reply:
Sounds good!